src/db/sysdb.c

```diff
@@ -2098,15 +2097,14 @@ bool sysdb_entry_attrs_diff(struct sysdb_ctx *sysdb,
     lret = ldb_search(sysdb->ldb, tmp_ctx, &res, entry_dn, LDB_SCOPE_BASE,
                       attrnames, NULL);
     if (lret != LDB_SUCCESS) {
-        ret = sysdb_error_to_errno(lret);
-        DEBUG(SSSDBG_MINOR_FAILURE, "Cannot search sysdb: %d\n", ret);
+        DEBUG(SSSDBG_MINOR_FAILURE, "Cannot search sysdb: %d\n",
+              sysdb_error_to_errno(lret));
         goto done;
     }
 
     if (res->count == 0) {
         return true;
     } else if (res->count != 1) {
```

Comment:
I was thinking about this one too. Maybe change it to something like that:
    if (res->count == 0) {
        return true;
    }
    if (res->count == 1) {
        differs = sysdb_ldb_msg_difference(entry_dn, res->msgs[0], new_entry_msg);
    }

Reply:
It is of course optional change, I just think it would make code look more flat and straightforward.

Reply:
I'm not against the change, but I'll not include it as it isn't related with the purpose of this PR.

Reply:
OK, I have already set ACCEPT flag after last review.